### PR TITLE
Remove vendor folder to make install easier

### PR DESCRIPTION
**Pull request by Remi-Tuy**
This make the 'make install' process easier :)

Issue Link: https://github.com/centrifuge/go-centrifuge/pull/676

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,6 +22,9 @@ help: ## Show this help message.
 	@echo 'targets:'
 	@egrep '^(.+)\:\ ##\ (.+)' ${MAKEFILE_LIST} | column -t -c 2 -s ':#'
 
+remove-vendors: ##remove old vendores folder
```
Comment (vimukthi-git) [inline Makefile]: spelling error -  vendores -> vendors

**Comment by vimukthi-git** [comment]
@Remi-Tuy We had a small discussion on this and thought that it would be nice to make this part of a `make clean` target. What do you think?

**Comment by Remi-Tuy** [comment]
@vimukthi-git agreed, please see update.

**Comment by vimukthi-git** [comment]
@Remi-Tuy there seems to be an issue in CI when building
screenshot 2019-01-24 at 15 36 22

**Pull request merged**